frontend/src/app/modules/common/autocomplete/autocomplete-select-decoration.component.ts

```diff
@@ -26,12 +26,13 @@
 // See docs/COPYRIGHT.rdoc for more details.
 // ++
 
-import {AfterViewInit, Component, ElementRef, OnInit, ViewChild} from '@angular/core';
-import {DynamicBootstrapper} from "core-app/globals/dynamic-bootstrapper";
+import {Component, ElementRef, OnInit, ViewChild} from '@angular/core';
 import {NgSelectComponent} from "@ng-select/ng-select";
 
 type SelectItem = { label:string, value:string, selected?:boolean };
 
+export const autocompleteSelectDecorationSelector = 'autocomplete-select-decoration';
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/be03a9ed2bd960efe9bf08e68a4088034cb73cd9/frontend/src/app/modules/common/autocomplete/autocomplete-select-decoration.component.ts#L35"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>